src/legacy/ui/public/exit_full_screen/_index.scss

```diff
@@ -1 +1 @@
-@import 'exit_full_screen_button';
+@import '../../../../plugins/kibana_react/public/exit_full_screen_button/exit_full_screen_button';
```

Comment:
This import should be pointing to an `_index.scss` file. Can you add a new one next to `_exit_full_screen_button.scss` which imports <-- that file. Then this line can point to that new `_index.scss` file.

Reply:
Done.